content/blog/2019/10/2019-10-28-devops-world-jenkins-world-san-francisco-in-living-colors.adoc

```diff
@@ -0,0 +1,105 @@
+- events
+- devopsworldjenkinsworld2019
+author: alyssat
+---
+link:https://www.cloudbees.com/devops-world/san-francisco[DevOps World | Jenkins World San Francisco] was August 12 - 15, 2019.
+The event was delivered in vivid colors starting with flowing banners hung from street lamp posts to the big screens in breakout rooms, to the expo hall.
+The energy and enthusiasm in the Moscone convention center made the colors even more vibrant, thanks to the people attending the conference.
+
+Here's a recap of the conference in pictures:
+
+.DevOps World | Jenkins World 2019 - San Francisco
+image:/images/post-images/jenkinsworld2019/1D5_2235.jpg[width=800]
+
+## Keynote - Evolution of the Continuous Delivery Foundation
+
+Tracy Miranda opened the keynote explaining the evolution of the Continuous Delivery Foundation.
+
+.Keynote - Evolution of the Continuous Delivery Foundation
+image:/images/post-images/jenkinsworld2019/1D5_0437.jpg[width=800]
+
+## Influencers, Creators, and Members
+
+The influencers, creators, and members of the CD Foundation: Tracy Miranda (far left),
+Andy Glover (Netflix), Tara Hernandez (Google), Chris Aniszczyk (Linux Foundation), Dave Stanke (Google), Kohsuke Kawaguchi (Jenkins creator), Jayne Groll (DevOps Institute), James Strachan (Jenkins X creator).
+“We want to help set Jenkins up for success, into the next decade”, Tyler Croy (not in picture).
+
+.Influencers, Creators, and Members of the Continuous Delivery Foundation
+image:/images/post-images/jenkinsworld2019/1D5_1310.jpg[width=800]
+
+## Contributor Summit
+
+The inaugural Continuous Delivery Foundation Contributor Summit and it was a full house!
+
+.Continous Delivery Foudnation Contributor Summit
+image:/images/post-images/jenkinsworld2019/IMG_8264.jpg[width=400]
+
+## 15 Years of Jenkins
+
+A remarkable milestone for the Jenkins project, a celebration of Jenkins turning 15...cake included!
+
+.Fifteen Years of Jenkins
+image:/images/post-images/jenkinsworld2019/1D5_0614.jpg[width=400]
+
+## Bee Diverse Luncheon
+
+Interactive and engaging luncheon celebrating diversity
+
+.Bee Diverse Luncheon Entrance
+image:/images/post-images/jenkinsworld2019/1D5_5576.jpg[width=400]
+
+.Bee Diverse Luncheon Leading Voices
+image:/images/post-images/jenkinsworld2019/1D5_5606.jpg[width=400]
+
+.Bee Diverse Luncheon Group Discussions
+image:/images/post-images/jenkinsworld2019/1D5_5682.jpg[width=400]
+
+## Jenkins Contributors and Experts
+
+Jenkins contributors and experts on hand to educate and share lightning talks and provide one on one Jenkins support.
+
+.Jenkins Lightning Talks
+image:/images/post-images/jenkinsworld2019/1D5_3207.jpg[]
+
+.Jenkins Experts Answering Questions
+image:/images/post-images/jenkinsworld2019/1D5_2953.jpg[width=400]
+
+.Jenkins Experts Discussing and Helping
+image:/images/post-images/jenkinsworld2019/IMG_8278.jpg[width=400]
+
+.Jenkins Experts Gathered
+image:/images/post-images/jenkinsworld2019/1D5_3573.jpg[width=400]
+
+# DevOps Superheroes
```

Comment:
First level heading? Not sure about it....

Reply:
Thanks for catching that.  I've rebased to simplify the changes into two commits and force pushed.